tests/analyze.test/t09_01.req.out

```diff
@@ -10,12 +10,16 @@
 (out='Analyze completed table t9')
 (out='Analyze table 't9' is complete')
 [exec procedure sys.cmd.analyze("t9")] rc 0
```

Comment:
The implementation of the 'analyze_empty_tables' tunable has been simplified slightly and it results in a different result for this test.